Shared/Extensions/DispatchQue+Extensions.swift

```diff
@@ -93,6 +93,7 @@ extension DispatchQueue {
     return work
 }
 
+@available(*, deprecated, message: "Use Task.onMainActor")
```

Comment:
We'll want to get rid of this function eventually and just run things on the MainActor.